README.md

```diff
@@ -1,120 +1,31 @@
 # Conversational Toolkit
 
-The Conversational Toolkit is a web application that provides a simple prototyping environment for voice-driven conversational AI agents.
+The Conversational Toolkit is a Web application that provides a simple prototyping environment for voice-driven conversational AI agents.
 
 The toolkit fronts a language model with automatic speech recognition (ASR) using the WebSpeech API, and text-to-speech (TTS) using the Vertex speech API. It is designed to be modular so that new use cases can be quickly added with well-encapsulated code changes.
 
-## Local Development
+## Instructions
 
-### Start backend server:
-To run the server locally, you must first install the Google Cloud CLI and set up your default project and application default credentials.
+| Documentation                                           | Description                                     |
+| --------------------------------------------------------| ----------------------------------------------- |
+| [Getting Started](./docs/getting-started.md)            | How to get started, run locally and on cloud    |
+| [Adding a new Agent](./docs/adding-a-new-agent.md) |    | How to build and test a new agent               |
```

Comment:
nit: extra `|` is preventing the row from being displayed correctly: 
<img width="706" alt="Screenshot 2024-07-08 at 11 52 51 AM" src="https://github.com/adamconnors/conversational-testbed/assets/2132122/72e5eb94-9e5c-4c84-9d98-3c73ab57edf6">